README.md

```diff
@@ -1,3 +1,98 @@
+# Hub V2
+
+Write new APIs here
+
+---
+## Getting Started
+
+
+## Create Dataset
+
+## Import Dataset
+
+## Train using hub Dataset
+
+## Train using hub Dataset on Neuro
+
+
+## Folder Structure:
```

Comment:
you should not put folder structure in the README. it isn't formatted well & it's redundant (the folder structure can be found in the hub folder).

Reply:
i still think the folder format in `hub/` is not intuitive. that, and as @benchislett was saying we should not lay this out beforehand, but instead let people working on the specific parts put their code where they think it fits & during the review process we can suggest locations. laying it out like this just makes it messy